spec/controllers/spree/admin/orders_controller_spec.rb

```diff
@@ -57,7 +57,7 @@
 
         it "updates fees and taxes and redirects to order details page" do
           expect(order).to receive(:recreate_all_fees!)
-          expect(order).to receive(:create_tax_charge!)
+          expect(order).to receive(:create_tax_charge!).at_least :once
```

Comment:
I puzzled over this a bit, but it looks like this happens in relation to what the test _setup_ is doing as opposed to what happens in the test itself. Basically before the test even starts (or when the order object is called and lazy-instantiated) the process of created a completed order (`:completed_order_with_totals`) is running through various bits of logic in  a sequence like adding line items and creating shipments and setting the shipping address, etc etc, and some of that is doing things that are triggering tax-related callbacks because the order total is changing or the tax address is changing.